src/eredis_client.erl

```diff
@@ -97,6 +95,8 @@ init([Host, Port, Database, Password, ReconnectSleep, ConnectTimeout, Options])
                    sentinel = Sentinel},
     case connect(State, Options) of
         {ok, NewState} ->
+            erlang:put(options, Options),
```

Comment:
it's safer (for future changes) to have these two lines in the beginning of init/1 ?